### PR TITLE
Clear console on Ctrl+L key binding

### PR DESCRIPTION
**Pull request by ar-maged**
- Clear console without having to click on the "Clear" button

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -5,3 +5,9 @@ $('[data-action="run"]').click(function() {
 $('[data-action="clear-console"]').click(function() {
     window.clearConsole();
 });
+
+$('#console').bind('keydown', 'ctrl+l', function() {
+    window.clearConsole();
+});
```
Comment (abkreno) [inline js/main.js]: does this handle `cmd+l` for mac users?

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "bootstrap": "3.3.7",
     "jquery": "3.2.1",
+    "jquery.hotkeys": "^0.1.0",
     "moment": "^2.18.1",
```
Comment (abkreno) [inline package.json]: please use exact versions (i forgot to do so in moment too), and bump patch

File: js/main.js (hunk 1 above)
Comment (ar-maged) [reply js/main.js]: Mac users also clear their terminals/consoles via Ctrl+L ✅

Comment (abkreno) [reply js/main.js]: okay perfect :+1: ( **PS:** i have never been a mac user before :joy: )

Comment (ar-maged) [reply js/main.js]: 😂

File: package.json (hunk 2 above)
Comment (ar-maged) [reply package.json]: Done 👍

File: js/main.js (hunk 1 above)
Comment (desoukya) [reply js/main.js]: You guys are missing out on a lot :see_no_evil:

fyi, to clear terminal, mac users use command-k (`⌘ + K`)

Comment (abkreno) [reply js/main.js]: :see_no_evil: :see_no_evil: :see_no_evil:

Comment (ar-maged) [reply js/main.js]: @desoukya I've been using a Mac for 3 months now, and I've been clearing the terminal using Ctrl+L the entire time - I thought it was the only key binding for this 😂 Thanks for the tip! 🙈

Comment (desoukya) [reply js/main.js]: I'm glad to hear you have converted to the light side :tada:

I'm glad to hear CTRL-L works! Maybe I have a custom mapping on my machine. 

Thanks for your contribution; it's always nice to see a familiar face :slightly_smiling_face:

**Pull request merged**